multipaz-barcodes/src/androidInstrumentedTest/kotlin/org/multipaz/barcodes/BarcodeScannerTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.multipaz.barcodes
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import kotlinx.coroutines.test.runTest
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.junit.Test
+import org.multipaz.compose.decodeImage
+import org.multipaz.multipaz_barcodes.generated.resources.Res
+import kotlin.test.assertEquals
+
+
+class BarcodeScannerTests {
+    @OptIn(ExperimentalResourceApi::class, ExperimentalTestApi::class)
+    @Test
+    fun testBarcodeScanner() = runTest {
```

Comment:
We should make this three tests instead of one. Then failures in early tests won't hide failures in later ones, and it may be easier to tell what failed from just the test name.

Reply:
Good idea, doing that.